base.py

```diff
@@ -190,9 +191,22 @@ def __init__(self, addr, username, password, create_test_user=True,
     def __del__(self):
         self.browser.quit()
 
+    def _cleanup_string(self, string):
+        tags = re.compile('<.*?>')
+        return re.sub(tags, '', string).strip()
+
+    def _to_typographic_punctuation(self, string):
+        command = "echo \"{0}\" | marked --smartypants".format(string)
```

Comment:
Why you need `echo` here instead of directly passing the argument to the command?